Formula/verapdf-apps.rb

```diff
@@ -0,0 +1,28 @@
+class VerapdfApps < Formula
+  desc "Open-source industry-supported PDF/A validation"
+  homepage "https://verapdf.org/home/"
+  url "https://github.com/veraPDF/veraPDF-apps/archive/refs/tags/v1.21.4.tar.gz"
+  sha256 "b926c397f98c03c506e486593e0224f4250c5329ed353a280e4dcfa26ca09b68"
+  license any_of: ["GPL-3.0-or-later", "MPL-2.0"]
+  head "https://github.com/veraPDF/veraPDF-apps.git", branch: "integration"
+
+  depends_on "maven" => :build
+  depends_on "openjdk"
+
+  def install
+    system "mvn", "clean", "install"
+
+    installer_file = Pathname.glob("installer/target/verapdf-izpack-installer-*.jar").first
+    system "java", "-DINSTALL_PATH=#{libexec}", "-jar", installer_file, "-options-system"
+
+    bin.install libexec/"verapdf", libexec/"verapdf-gui"
+    bin.env_script_all_files libexec, JAVA_HOME: Language::Java.overridable_java_home_env
```

Comment:
```suggestion
    bin.env_script_all_files libexec, Language::Java.overridable_java_home_env
```
Oops, my mistake

Reply:
Is this better than the following?
```
bin.env_script_all_files libexec, JAVA_HOME: Formula["openjdk"].opt_prefix
```
(I see that in various other formulas.)

Reply:
Yes, it's better.

Reply:
Pushed already. It looks like the only issue now is the exit code for the out-of-memory test, which is wrong on arm64 for some reason.